app/modules/asset_groups/schemas.py

```diff
@@ -27,11 +27,18 @@ class DetailedAssetGroupSightingSchema(BaseAssetGroupSightingSchema):
     Detailed Asset_group_sighting schema exposes all useful fields.
     """
 
+    completion = base_fields.Function(lambda obj: obj.get_completion())
```

Comment:
@karenc I found this example online
https://marshmallow.readthedocs.io/en/stable/examples.html
the magic of how this lambda allows you to get at the object to call the method is a bit of a mystery 
I have no idea if this is a really clever idea or a bad hack :)
Any comments?